src/components/pages/get-a-quote/steps.svelte

```diff
@@ -26,7 +28,16 @@
         'lg:border-l lg:border-dashed'
       )}
     >
-      <div class="absolute -left-px -top-32 hidden h-32 border-l lg:block" />
+      <div
+        class="absolute -left-px -top-32 hidden h-32 border-l lg:block"
+        style={variant === 'block'
+          ? `border-image: linear-gradient( to top, ${
+              $theme === 'light' ? '#e8e8e8' : '#2E2E2E'
+            } 0%, ${
+              $theme === 'light' ? 'rgba(247, 247, 247, 0.00)' : 'rgba(46, 46, 46, 0.00)'
+            } ) 1 100%;`
+          : ''}
```

Comment:
don't we have tokens for this that react to the theme? I don't agree with having lost hard coded colors